llvm/test/CodeGen/WebAssembly/signext-zeroext-callsite.ll

```diff
@@ -0,0 +1,125 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -O0 | FileCheck %s
```

Comment:
Is there a test that covers this behavior for DAG ISel? Maybe it would make sense to add test expectations for both here?

Reply:
I added the -fast-isel=false case to check the case when the regular DAG ISel path is taken.